src/components/App.js

```diff
@@ -5,14 +5,16 @@ import {
   Route,
   Redirect,
 } from 'react-router-dom';
-import { HOME_PATH } from '../config/paths';
-import Home from './main/Home';
+import { buildMainPath } from '../config/paths';
+import MainScreen from './main/MainScreen';
 
 export const App = () => (
   <Router>
     <Switch>
-      <Route path={HOME_PATH} exact component={Home} />
-      <Redirect to={HOME_PATH} />
+      <Route path="/" exact component={MainScreen} />
+      <Route path="/:rootId/:id" exact component={MainScreen} />
+      <Route path="/:rootId" exact component={MainScreen} />
```

Comment:
Why are the three the same screen?

Reply:
We have one screen only (yet) depending on the root item id and its children id. Currently `/` shows nothing (Item not defined screen). `/roodId` displays the root item's first level children, and `/rootId/id` selects directly the item, which displays this item's first level children.